VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+* **Hack bounty:** This is an incentive for the community to hack validators. There will be bounties proportionate to the size of the validator, so that a validator becomes a bigger target as its stake grows.
+* **UI warning:** Users will be warned by the Cosmos UI if they want to delegate to a validator that already has a significant amount of staking power.
+
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+It is highly probable that they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block provisions and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called `photons`.
+
+
+## Technical requirements 
+
+### How can I join the testnet?
+
+The Testnet is a great environment to test your validator setup before launch.
+
+We view testnet participation as a great way to signal to the community that you are ready and able to operate a validator. You can find all relevant information about the testnet and more here : https://cosmos.network/validators
+
+### Is there a faucet?
+
+If you want to obtain coins, you can do so by using this faucet (maintained by community) : https://www.cosmosvalidators.com/
+
+
+### What are hardware requirements?
+
+Validators should expect to provision one or more data center locations with redundant power, networking, firewalls, HSMs and servers.
+We expect that a modest level of hardware specifications will be needed initially and that they might rise as network use increases. Participating in the testnet is the best way to learn more.
+ 
+### What are software requirements?
+
+In addition to running a Cosmos Hub node, validators should develop monitoring, alerting and management solutions.
+
+### What are bandwidth requirements?
+
+The Cosmos network has the capacity for very high throughput relative to chains like Ethereum or Bitcoin.
+We recommend that the data center nodes only connect to trusted full-nodes in the cloud or other validators that know each other socially. This relieves the data center node from the burden of mitigating denial-of-service attacks.
+Ultimately, as the network becomes more heavily used, multigigabyte per day bandwidth is very realistic.
+
+### What does running a validator imply in terms of logistics? 
+
+A successful validator operation will require the efforts of multiple highly skilled individuals and continuous operational attention. This will be considerably more involved than running a bitcoin miner for instance.
+
+### How to handle key management?
+
+Validators should expect to run an HSM that supports ed25519 keys. Here are the current options:
+ * YubiHSM 2
+ * Ledger Nano S
+ * Potential options
+ * Ledger BOLOS SGX enclave
+ * Thales nCypher support
```

Comment:
Can we correct this to "Thales nShield".

Reply:
sure